src/main/resources/messages.properties

```diff
@@ -1077,3 +1076,61 @@ error.missing-income=Be sure to provide your job income before taxes.
+add-document-types.doc-type-label.income.other-income=Other Income
+add-document-types.doc-type-label.expenses.bills=Bills
+add-document-types.doc-type-label.medical.medical-info=Medical Information
+add-document-types.doc-type-label.resources.banking-info=Banking Information
+add-document-types.doc-type-label.legal.marriage-license=Marriage License
+add-document-types.doc-type-label.legal.divorce-decree=Divorce Decree
+add-document-types.doc-type-label.legal.court-order=Court Order
+add-document-types.doc-type-label.paternity.ack-of-paternity=Acknowledgement of Paternity
+add-document-types.doc-type-label.correspondence.other=Other
+
+# doc upload intro
+doc-upload-intro.title=Doc upload intro
+doc-upload-intro.header=Your application is complete! Would you like to upload documents?
+doc-upload-intro.subtext=This step is optional. You may need to give documents later on in your application process.
+doc-upload-intro.continue=Continue to upload docs
+doc-upload-intro.skip=Not right now
+
+# doc upload signpost
+doc-upload-signpost.title=Doc upload signpost
+doc-upload-signpost.step=Step 6 out of 6
+doc-upload-signpost.header=Uploading documents
+doc-upload-signpost.well-ask-about=We'll tell you
+doc-upload-signpost.how-to-add-docs=How to add docs
+doc-upload-signpost.which-docs=Which docs you can add
+doc-upload-signpost.how-to-submit=How to submit now or later
+
+doc-upload-instructions.optional=Adding documents to your application is <b>optional</b>.
+
+# doc upload recommendations
+doc-upload-recommendations.title=Doc upload recommendations
+doc-upload-recommendations.header=Submit any documents that you have on hand
+doc-upload-recommendations.subtext=You have 30 days to submit your verification documents - but the sooner you submit them, the sooner we can determine your eligibility. If you don't have these documents now, you can upload them after you submit this application.
+doc-upload-recommendations.recommended=Recommended documents
+doc-upload-recommendations.income=Proof of Income
+doc-upload-recommendations.income-content=Tax documents from 2020 or 2021 (such as Form 1040), or wage documents (such as pay stubs, W-2) for all adults in the household that earn income.
+doc-upload-recommendations.residency=Proof of Residency
+doc-upload-recommendations.residency-content=Current official document with your name and address such as a valid government ID, housing documents (rent receipts, lease, or mortgage statements), or a recent utility bill.
+doc-upload-recommendations.identity=Proof of Identity
+doc-upload-recommendations.identity-conent=A government-issued photo ID such as a driver's license, identity card, passport or Social Security card.
+doc-upload-recommendations.types-of-files=Types of files
+doc-upload-recommendations.types-of-files-content=We accept .jpeg, .jpg, .png, .pdf, .bmp, .gif, .doc, .docx, .odt, .ods, or .odp files. Each file is limited to 20MB. The maximum number of files is 20.
```

Comment:
Let's paramaterize the `20MB` - or just hardcode it to `5MB` for now. Aside, do we have this limit now that we're not using the RPA bot??